<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDataWindow</class>
 <widget class="QDialog" name="MyDataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dane</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>285</width>
     <height>343</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="myDataLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Moje dane</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hasło</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordOutput"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Imię</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameOutput"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="surnameLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nazwisko</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="surnameOutput"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="emailLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailOutput"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="phoneNumberLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nr telefonu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phoneNumberOutput"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="peselLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pesel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="peselOutput"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="dateOfBirthLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Data urodzenia</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dateOfBirthOutput"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="changeButton">
        <property name="text">
         <string>Zmień</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Wróć</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
